src/main/java/defusion/Country.java

```diff
@@ -0,0 +1,51 @@
+package defusion;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Country {
+    public final String name;
+    @Getter
+    private final Coordinate lowerLeftPosition;
+    @Getter
+    private final Coordinate upperRightPosition;
+    @Getter
+    private final List<City> cities = new ArrayList<>();
+
+    @Getter
+    @Setter
+    private int completeDay = -1;
```

Comment:
Use some descriptive static constant instead of -1

Reply:
ready